packages/wonder-blocks-modal/components/modal-close-button.js

```diff
@@ -0,0 +1,69 @@
+// @flow
```

Comment:
Can you add a `TODO` that this should be replace with an `IconButton` when that component is ready and that this is just a temporary component.

Reply:
My intent was to keep this component, and use `IconButton` internally. That way, we can more easily keep the button in sync across modals, including attributes like the ARIA label. How does that sound to you?

Reply:
Sounds good.